internal/server/keyRingEndpoint.go

```diff
@@ -130,6 +155,16 @@ func (kre *KeyRingEndpoint) getLoadedPrivateKeys(w http.ResponseWriter, r *http.
 	LogExit(log, r, 200, n)
 }
 
+// Delete Private Key godoc
+// @id kre-del-private-key
+// @tags Key Ring, Key Store
+// @Summary Delete a private key from the server
```

Comment:
```Deletes a GPG Private Key from the server```

ahhh from the server é o próprio Chevron? Se for isso eu tiraria o `from the server`.